lib/Transforms/BufferPlacement/HandshakePlaceBuffers.cpp

```diff
@@ -181,10 +181,7 @@ LogicalResult HandshakePlaceBuffersPass::placeUsingMILP() {
   NameAnalysis &nameAnalysis = getAnalysis<NameAnalysis>();
   if (!nameAnalysis.isAnalysisValid())
     return failure();
-  if (!nameAnalysis.areAllOpsNamed()) {
-    if (failed(nameAnalysis.walk(NameAnalysis::UnnamedBehavior::NAME)))
-      return failure();
-  }
```

Comment:
I think this is done at https://github.com/EPFL-LAP/dynamatic/blob/fca6f2208693bdee63f6df2a0f31519eb15b5518/lib/Transforms/BufferPlacement/HandshakePlaceBuffers.cpp#L145